pages/proposals/all.vue

```diff
@@ -48,16 +40,6 @@
               {{ option }}
             </option>
           </select>
-
-          <select
-            v-model="selectedStatus"
```

Comment:
why removed?

Reply:
to match design:
![image](https://github.com/user-attachments/assets/260c2074-444e-4074-af0c-26e8add39e8f)